### PR TITLE
Feat tts elevenlabs

### PR DESCRIPTION
**Pull request by sunshinexcode**

### CODE REVIEW

File: agents/addon/extension/elevenlabs_tts/manifest.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    "type": "extension",
+    "name": "elevenlabs_tts",
+    "version": "0.1.0",
+    "language": "go",
+    "dependencies": [
+        {
+            "type": "system",
+            "name": "rte_runtime",
+            "version": "0.1.0"
+        }
```
Comment (wangyoucao577) [inline agents/addon/extension/elevenlabs_tts/manifest.json:11]: and also `rte_runtime_go`

**Review by zhangtao1104 — APPROVED** [review]

File: agents/addon/extension/elevenlabs_tts/elevenlabs_tts_extension.go [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+					break
+				}
+				if err != nil {
+					slog.Error(fmt.Sprintf("read PCM stream failed, err: %v", err), logTag)
+					break
+				}
+
+				if n != pcmFrameSize {
+					slog.Warn(fmt.Sprintf("the number of bytes read is [%d] inconsistent with PCM frame size", n), logTag)
+					continue
```
Comment (wangyoucao577) [inline agents/addon/extension/elevenlabs_tts/elevenlabs_tts_extension.go:223]: These data still could be used, just fill in blank for the rest. It's common at the end.

File: agents/addon/extension/elevenlabs_tts/elevenlabs_tts_extension.go [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+	bytesPerSample := int32(2)
+	pcmFrameSize := int(samplesPer10ms * channel * bytesPerSample) // per 10ms
+
+	// init chan
+	textChan = make(chan *message, textChanMax)
+
+	go func() {
+		slog.Info("process textChan", logTag)
+
+		for msg := range textChan {
```
Comment (wangyoucao577) [inline agents/addon/extension/elevenlabs_tts/elevenlabs_tts_extension.go:173]: It's better to handle `OnStop` gracefully.

File: agents/addon/extension/elevenlabs_tts/elevenlabs_tts_extension.go [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+				if err != nil {
+					slog.Error(fmt.Sprintf("read PCM stream failed, err: %v", err), logTag)
+					break
+				}
+
+				if n != pcmFrameSize {
+					slog.Warn(fmt.Sprintf("the number of bytes read is [%d] inconsistent with PCM frame size", n), logTag)
+					continue
+				}
+
```
Comment (wangyoucao577) [inline agents/addon/extension/elevenlabs_tts/elevenlabs_tts_extension.go:225]: It's better to measure `first_frame_latency` here.

**Pull request merged**